shared/common-adapters/markdown-react.js

```diff
@@ -0,0 +1,413 @@
+// @flow
```

Comment:
i think we should make a folder, common-adapters/markdown and put all this stuff in there

Reply:
yeah for sure, I wanted to get this up and running before restructuring. Doing that now